package.json

```diff
@@ -69,7 +69,7 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.3.1",
-    "storybook": "^8.2.9",
+    "storybook": "^7.6.17",
```

Comment:
Storybook v8+ breaks our current implementation so downgrading in order to avoid that issue.